etl/scripts-ccao-data-warehouse-us-east-1/ccao/ccao-commercial_valuation.R

```diff
@@ -102,16 +109,14 @@ renames <- c(
 )
 
 # Compile a filtered list of excel workbooks and worksheets to ingest ----
-temp <- list.files(
+list.files(
   "G:/1st Pass spreadsheets",
   pattern = "[0-9]{4} Valuation",
   full.names = TRUE
 ) %>%
   file.path("PublicVersions") %>%
-  list.files(ignore.case = TRUE, full.names = TRUE) %>%
-  # Ignore files that are currently open
+  list.files(pattern = ".xlsx", full.names = TRUE, recursive = TRUE) %>%
```

Comment:
File tree is different this year than it was previously, this is a more flexible way to find the files we need.